apps/web/app/(use-page-wrapper)/settings/(settings-layout)/SettingsLayoutAppDirClient.tsx

```diff
@@ -449,11 +455,15 @@ const SettingsSidebarContainer = ({
     }[]
   >();
   const session = useSession();
-  const { data } = trpc.viewer.delegationCredential.check.useQuery(undefined, {
```

Comment:
We could delete the route as well. Doesn't seem to be used elsewhere

Reply:
The check route is also used in middleware inside the delegation router

Reply:
Oh its not ignore me